Archipelago.cpp

```diff
@@ -225,6 +225,53 @@ void AP_Start() {
     }
 }
 
+void AP_Shutdown() {
+    webSocket.stop();
```

Comment:
i guess webSocket.stop() should only be called if multiworld == true;

Reply:
Correct

Reply:
I'm curious, why do you need that?